requirements.txt

```
@@ -1,37 +1,37 @@
 anyio==3.7.0
-blis==0.7.9
+blis==1.3.0
 catalogue==2.0.8
```

```suggestion
catalogue==2.0.10
```

If I get it right, there seems to be a problem with this package and Python 3.12. Updating the version worked for me.